pkg/Makefile.am

```diff
@@ -26,6 +26,7 @@ metainfo_DATA = pkg/sosreport/org.cockpit-project.cockpit-sosreport.metainfo.xml
 		pkg/kdump/org.cockpit-project.cockpit-kdump.metainfo.xml \
 		pkg/selinux/org.cockpit-project.cockpit-selinux.metainfo.xml \
 		pkg/machines/org.cockpit-project.cockpit-machines.metainfo.xml \
+		pkg/storaged/org.cockpit-project.cockpit-storaged.metainfo.xml \
```

Comment:
Can you please rm the new file in the "when not building optional packages, remove their files" section, too?

You can test with tools/make-rpm when setting `build_optional` to 0 in the spec. I bet it currently fails with an "this file has not been packaged anywhere *mimimi*"